[ECOINT-283] Rebrand integration from Instabug to Luciq (#2808)
DataDog/integrations-extras: instabug/CHANGELOG.md

# CHANGELOG - Luciq

> ## 2.1.0 / 2025-10-10

Reviewer: ```suggestion
## 3.1.0 / 2025-10-10
```

# CHANGELOG - Luciq

> ## 2.1.0 / 2025-09-24

Reviewer: ```suggestion
## 2.1.0 / 2025-10-10
```

***Changed***:

> * Rebranded integration from "Instabug" to "Luciq" 
> * Updated company information and contact details to reflect Luciq branding
> * Updated support email to support@luciq.ai and sales email to success@luciq.ai
> * Updated homepage URL to https://www.luciq.ai/
> * Refreshed documentation and README to reflect Luciq branding throughout
> 
> ***Added***:
> 
> * New media assets with Luciq branding and updated screenshots
> * Enhanced media gallery with comprehensive feature showcases including AI-powered smart resolution

Reviewer: ```suggestion
* Rebranded integration from Instabug to Luciq, updating company info, contact details, emails, homepage, and documentation accordingly
```
